### ISSUE TITLE
Colored filenames when using the tags command

### ISSUE THREAD

**Issue by fturco** [oniony/TMSU#80]
I usually display tags for my PDF e-books with the following command:

    tmsu tags *.pdf

It would be easier for me to check the output if filenames were highlighted with a certain color to differentiate them from tags and values. I tried the --color=always but it doesn't do that.

**Comment by oniony** [comment]
Thanks for raising this. Colour is already used in the `tags` output to highlight implicitly and explicitly implied tags in cyan and yellow respectively. I did consider highlighting more elements in colour but it gets a bit too much if everything is coloured. It also gets a bit tricky to pick colours that work well on both dark and light colour schemes: go to far and I would need to add a way to customize the colours used.

I could, perhaps, use bold for the filenames though.

Another option would be to tabulate the output (on terminals) so that the tag names all start from the same column?

Let me know what you think.

**Comment by fturco** [comment]
In my case I have filenames that can be quite long (e-book titles), so I don't think tabulating output will be ok. Bold characters for filenames may be a good idea instead IMHO.

**Comment by oniony** [comment]
I think the best way forward with this would be customizable colours that would be stored in the settings. However I think it would make more sense for these to be stored in user settings (e.g. `~/.tmsu/colors`) rather than in the database settings, as I imagine one would want consistent colours between databases and one wouldn't want to necessarily impose one's colours on someone else if one were to share a database.

**Comment by fturco** [comment]
I agree that color settings should go in a configuration file as opposed to databases.

**Comment by nkh** [comment]
@ghost pipe through https://github.com/sharkdp/lscolors
